twitter-ads.gemspec

```diff
@@ -8,14 +8,20 @@ Gem::Specification.new do |s|
   s.version     = TwitterAds::VERSION
   s.platform    = Gem::Platform::RUBY
   s.licenses    = ['MIT']
-  s.authors     = ['Brandon Black', 'John Babich', 'Jacob Petrie']
+  s.authors     = [
+    'John Babich',
+    'Tushar Bhushan',
+    'Juan Shishido',
+    'Thomas Osowski',
+    'Shohei Maeda'
```

Comment:
We can add Brandon Black if we want to include the original author of this one...(not sure what standard practice is)

Reply:
We removed him from Python SDK as well so I'll follow this.